app/src/main/java/com/example/roome/RoommateSearcherSetProfileActivity.java

```diff
@@ -1,18 +1,25 @@
 package com.example.roome;
```

Comment:
why you didn't do it in the class EditProfileRoommateSearcher that is already created?

Reply:
Because it's not the same thing. you are right that I need to change this name of activity because we decided to make it at an apartment setup. 

I'll rename the activity to roommateSearcherSetApartmentInfo